Corona-Warn-App/src/main/res/values-de/dcc_ticketing_strings.xml

```diff
@@ -8,6 +8,10 @@
     <string name="dcc_ticketing_error_service_provider_error">Es trat ein Fehler bei der Verbindung mit dem Anbieter %1$s auf. Bitte kontaktieren Sie den Anbieter wenn das Problem weiterhin besteht.</string>
     <!-- XTXT: Try again error message-->
     <string name="dcc_ticketing_error_try_again">Es trat ein Fehler bei der Verarbeitung der Daten auf. Bitte versuchen Sie es später erneut.</string>
+    <!-- XTXT: Validation service min version message-->
+    <string name="dcc_ticketing_error_min_version">In der aktuellen Version ihrer Corona-Warn-App wird die Tickerüberprüfung nicht mehr unterstützt. Bitte aktualisieren Sie Ihre App und versuchen Sie die Ticketüberprüfung erneut.</string>
```

Comment:
Typos: Change to "In der aktuellen Version Ihrer Corona-Warn-App wird die Ticketüberprüfung..."